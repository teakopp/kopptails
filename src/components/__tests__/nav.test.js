import renderer from 'react-test-renderer';
import Nav from "../nav";

test("renders nav component", () => {
  const tree = renderer
    .create(<Nav title="Test" subtitle="Subtest"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
