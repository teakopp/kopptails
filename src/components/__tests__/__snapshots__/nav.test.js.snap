// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders nav component 1`] = `
<div
  className="Nav"
>
  <header
    className="nav-title"
  >
    <div>
      Test
    </div>
  </header>
  <div
    className="nav-container"
  >
    <div
      className="nav-item"
    >
      <button
        className="nav-clear-button"
      >
        <img
          alt="search"
          className="nav-menu"
          src="bars-solid.svg"
        />
      </button>
    </div>
    <div
      className="nav-item"
    >
      <div
        className="nav-location"
      >
        <div>
          Subtest
        </div>
      </div>
    </div>
    <div
      className="nav-item"
    >
      <button
        className="nav-clear-button"
      >
        <img
          alt="search"
          className="nav-search"
          src="magnifying-glass-solid.svg"
        />
      </button>
    </div>
  </div>
  <div
    className="line"
  />
  <div />
</div>
`;
